pkg/pod/entrypoint.go

```diff
@@ -42,7 +42,10 @@ const (
 	entrypointBinary = binDir + "/entrypoint"
 
 	runVolumeName = "tekton-internal-run"
-	runDir        = "/tekton/run"
+
+	// RunDir is the directory in the containers where the step writes its
+	// post files to let the next step know to continue.
```

Comment:
nit: (since this directory contains more than just post files)

```suggestion
	// RunDir is the directory that contains runtime variable data for TaskRuns.
	// This includes files for handling container ordering, exit status codes, and more.
	// See [https://github.com/tektoncd/pipeline/blob/main/docs/developers/taskruns.md#tekton]
	// for more details.
```

Reply:
thanks. I added it.